src/components/PostList/PostList.jsx

```diff
@@ -1 +1,11 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
```

Comment:
redundant fragment
If a mentor points out a mistake, it means that you need to fix ALL occurrences of the mistake, not only in the place where the comment was left.